src/sentry/conf/server.py

```diff
@@ -1601,8 +1600,6 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
     "organizations:integrations-open-pr-comment-js": False,
     # Enable Opsgenie integration
     "organizations:integrations-opsgenie": True,
-    # Enable one-click migration from Opsgenie plugin
-    "organizations:integrations-opsgenie-migration": False,
```

Comment:
Should be safe in the same PR since the flag isn't used for any backend checks.

If the frontend is an old version, the button stays hidden (no change), if the frontend is up-to-date, the button is visible as expected. The API would work regardless, so no failure case grouping these changes.